.github/workflows/anari_sdk_ci.yml

```diff
@@ -28,6 +28,7 @@ jobs:
         -DBUILD_SHARED_LIBS=ON
         -DBUILD_EXAMPLES=ON
         -DBUILD_HELIDE_DEVICE=ON
+        -DBUILD_REMOTE_DEVICE=ON
```

Comment:
For now my preference would be to just test this on Ubuntu. To do that you should change this line to `-BUILD_REMOTE _DEVICE=${{ matrix.os == 'ubuntu-latest' }}`.